### PR TITLE
Update categories for new app update

### PR DESCRIPTION
**Pull request by joseph619**

### CODE REVIEW

**Comment by joseph619** [comment]
I updated the patch

File: src/.vuepress/components/Navigation.vue [hunk 1]
```diff
@@ -61,15 +61,17 @@ export default {
 				/* More */
 				downloaded_only: { text: "Downloaded only", icon: "cloud_off" },
 				download_queue: { text: "Download queue", icon: "get_app" },
-				source_migration: { text: "Source migration", icon: "compare_arrows" },
+				categories: { text: "Categories", icon: "label" }
```
Comment (arkon) [inline src/.vuepress/components/Navigation.vue]: Build fails because you missed a comma.

Comment (joseph619) [reply src/.vuepress/components/Navigation.vue]: Yep fixed

**Comment by arkon** [comment]
The build still fails.

You need to update the usages of these keys if you're changing them.

**Comment by joseph619** [comment]
Hmm, I guess I will put them back in for now then

**Comment by joseph619** [comment]
Should be fine now, build is no longer failing

File: src/.vuepress/components/Navigation.vue [hunk 2]
```diff
@@ -83,6 +83,10 @@ export default {
 				share: { text: "Share", icon: "share" },
 				webview: { text: "WebView", icon: "public" },
 				/* Browse */
+				button_latest: { text: "LATEST" },
+				button_browse: { text: "BROWSE" },
+				tab_sources: { text: "SOURCES" },
+				tab_extensions: { text: "EXTENSIONS" },
```
Comment (GHOSTBEAR) [inline src/.vuepress/components/Navigation.vue]: If you planned to remove these they are still in use, and you will need to remove the components that are using them to have a successful build. You can use the GitHub repository search function to find the uses if you don't use VSCode or something

Comment (joseph619) [reply src/.vuepress/components/Navigation.vue]: I would remove them later, just want to update the page with new content for now

**Comment by arkon** [comment]
I suspect the reference that the branch is diffing against might be messed up or something. Will need to clean stuff up at some point.

**Pull request merged**